doc/documentation.rst

```diff
@@ -9,7 +9,7 @@ Documentation
    :Release: |version|
    :Date: |today|
 
-Previous versions:  `0.12.0 <http://nipype.readthedocs.io/en/0.12.0/>`_ `0.11.0 <http://nipy.org/nipype/0.11.0>`_
+Previous versions:  `0.12.1 <http://nipype.readthedocs.io/en/0.12.1/>`_ `0.11.0 <http://nipy.org/nipype/0.11.0>`_
```

Comment:
yes - this should have been done after the `0.12.1` release but we missed it. and we use version in terms of minor element (11, 12, 13, etc.,.)

right after this release, when we switch back to the dev version this will become `0.13.0`